examples/chart/teleport-daemonset/README.md

```diff
@@ -0,0 +1,112 @@
+# Teleport
+
+[Gravitational Teleport](https://github.com/gravitational/teleport) is a modern SSH/Kubernetes API proxy server for
+remotely accessing clusters of Linux containers and servers via SSH, HTTPS, or Kubernetes API.
+
+## Introduction
+
+This chart deploys Teleport components to your cluster via a Kubernetes `Deployment`.
+
+By default this chart is configured as follows:
+
+- 1 replica
+- Record ssh/k8s exec and attach session to the `emptyDir` of the Teleport pod
+  - These sessions will also be stored on the root cluster, if used to access this Helm-configured cluster remotely
+- TLS is enabled by default on the Proxy
+  - The leaf cluster will generate its own self-signed certificates
```

Comment:
- An Enterprise Version of Teleport 

( Do we have a Community Helm Chart? )

Reply:
Added some clarification.

This Helm chart can actually run the OSS version very easily with a few small changes, but we don't explicitly provide instructions on how to do that (yet)